packages/core/database/src/metadata/relations.ts

```diff
@@ -84,21 +84,30 @@ const createOneToOne = (
   attributeName: string,
   attribute: Relation.OneToOne,
   meta: Meta,
-  metadata: Metadata
+  metadata: Metadata,
+  options: MetadataOptions
```

Comment:
It feels a bit weird having all those options, especially having meta, metadata and metadata options next to each others. but that's an issue for another day

Reply:
We could(/probably should) limit the number of arguments we can pass to a function in our eslint conf

Reply:
Mentioned in another comment; I agree, and my goal is to eliminate the options entirely, but once it was done, working, and tested, I didn't want to do another major refactoring last minute.